### DISCUSSION TITLE
Please indicate your favourite places to eat near NAIST.

### DISCUSSION THREAD

**Discussion by momoka-jp** [NAIST-SE/AttractingContributors#1187] [2024 Attraction Contributions]
https://github.com/momoka-jp/AttractingContributors/blob/main/README.md

**Comment by Idescrew** [comment]
I just made a PR on your repo.
Could you contribute to my repo? Thanks in advance.

https://github.com/Idescrew/AttractingContributors

**Comment by marshr00m** [comment]
I submitted a pull request. I would appreciate any additions to my list!
https://github.com/marshr00m/AttractingContributors-Where-would-you-like-to-travel

**Comment by masataka-ki** [comment]
I've sent you a pull request, so if you want, please pull request for me too!
https://github.com/masataka-ki/AttractingContributors-Please-tell-me-a-country-you-want-to-visit-in-the-future
